### PR TITLE
Update configuration and property key

### PR DESCRIPTION
**Pull request by LuQQiu**
Update configuration and property key in Alluxio documentation

### CODE REVIEW

File: dora/core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 1]
```diff
@@ -2625,7 +2625,7 @@ public static String getSyncMetricName(long mountId) {
           .build();
 
   public static final MetricKey NETTY_CHANNEL_COUNT =
-      new Builder("Netty.NettyChannelCount")
+      new Builder("Process.NettyChannelCount")
```
Comment (LuQQiu) [inline dora/core/common/src/main/java/alluxio/metrics/MetricKey.java:2628]: @JiamingMai PTAL, the original bin/alluxio docGen command complain about Netty is not one of the existing instance type
```
  public enum InstanceType {
    CLUSTER("Cluster"),
    SERVER("Server"),
    MASTER("Master"),
    WORKER("Worker"),
    SECURITY("Security"),
    JOB_MASTER("JobMaster"),
    JOB_WORKER("JobWorker"),
    PLUGIN("Plugin"),
    PROCESS("Process"),
    PROXY("Proxy"),
    CLIENT("Client"),
    FUSE("Fuse");

```

File: docs/_data/table/en/cluster-management-configuration.yml [hunk 2]
```diff
@@ -1,10 +0,0 @@
-alluxio.integration.master.resource.cpu:
```
Comment (Xenorith) [inline docs/_data/table/en/cluster-management-configuration.yml:1]: this file is empty now, delete it? as well as its reference in the property list tables

```
$ grep -r "cluster-management-configuration" .                                                                                                                                                [20:59:58]
./cn/reference/Properties-List.md:## 集群管理配置项 {#cluster-management-configuration}
./cn/reference/Properties-List.md:{% for item in site.data.table.cluster-management-configuration %}
./cn/reference/Properties-List.md:    <td>{{ site.data.table.cn.cluster-management-configuration[item.propertyName] }}</td>
./en/reference/Properties-List.md:{% for item in site.data.table.cluster-management-configuration %}
./en/reference/Properties-List.md:    <td>{{ site.data.table.en.cluster-management-configuration[item.propertyName] }}</td>
```

**Review by Xenorith — CHANGES_REQUESTED** [review]

**Review by Xenorith — APPROVED** [review]

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**